packages/yourloops/lib/patient/models/patient.model.ts

```diff
@@ -39,5 +39,6 @@ export interface Patient {
   metadata: PatientMetadata
   monitoring?: Monitoring
   teams: PatientTeam[]
+  currentTeam?: PatientTeam
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Is this the temporary hack that you talked about?

Reply:
Exactly bro 😉